CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.8.1
```

Comment:
```suggestion
## Unreleased
```
Generally we only assign a release number in the CHANGELOG when we're preparing the PR that actually bumps the version (normally in a separate PR).

Reply:
Good point. I kept this in a separate commit for this reason – but didn't update the corresponding `VERSION.rb` anyway, so this is a bit messy as is.